### DISCUSSION TITLE
generate-workspace-summary broken by new timestamps

### DISCUSSION THREAD

**Discussion by inhake2** [nvim-neorg/neorg#1085] [General]
I don’t fill a bug report as I’m not sure it is not intended and when the problem started. I don’t feel qualified to consider it as a bug myself as it’s just need some adjustment from the user, which is normal when using a fast moving project as neorg !

It may comes from a8f7a9eeef5c22eac626e7533eeee0ac9def72ad who kind of broke `generate-workspace-summary`, at least it’s linked to a change in the timestamp format that disturb the parser.

Most of my neorg files have categories in the metadata like :

```
…
categories : nftables
created: 2023-06-17
updated: 2023-06-17
```
or
```
…
categories : [
thesis
infrastructures
]
created: 2023-06-17
updated: 2023-06-17

```
`generate-workspace-summary` don’t sort links by categories anymore if the categorized files have the old timestamp in the metadata. All files are "uncategorised".

Updating the file (so the updated field) solves the problem. It works as intended with the following sample :

```
categories : nftables
created: 2023-06-17
updated: 2023-09-16T11:07:59+0100
```

@champignoom would be in a better position to judge if it’s need more investigation and devs attention/time, but I share it here as it may help other end-users facing the same problem, as it’s easy to fix by the users themselves : just re-save the considered files or update the updated timestamp with some shell command. New workspaces and files will use the new timestamp by default so the problem should not occur in the future.

Tell me if it’s worth a bug report or need some infos !

**Comment by austinbutler** [comment]
Even with the new dates it doesn't seem to work for me. Created https://github.com/nvim-neorg/neorg/issues/1108.

**Comment by inhake2** [comment]
Yes, sorry I didn’t update. It doesn’t work for me anymore, and I think I misunderstood what the problem is when opening this thread.

In fact, it doesn’t seem to be a timestamp problem. Categories are only taken into account once the file has been opened.

Thank you for opening an issue, I missed time to do it.
